pipeline/selection/g4OnSelection.rml

```diff
@@ -26,7 +26,7 @@
  
     <addProcess type="TRestEventSelectionProcess" name="evSelection" fileWithIDs="Run00001_NLDBD_Test_g4Analysis.root" conditions="g4Ana_totalEdep<2400 AND g4Ana_energyPrimary>1000" value="ON"  verboseLevel="info"/>
     
-    <addProcess type="TRestEventSelectionProcess" name="evSelection" fileWithIDs="IDs.txt" value="ON"  verboseLevel="info"/>
+    <!--<addProcess type="TRestEventSelectionProcess" name="evSelection" fileWithIDs="IDs.txt" value="ON"  verboseLevel="info"/>-->
```

Comment:
why this was commented out? It was not said at the PR description.

Reply:
It is mentioned that some pipelines are updated.

In principle I think adding two TRestEventSelectionProcess is not a good pipeline script. We shall run two indepent processes to test "condition selection" and "ids file selection" functionality.

Reply:
Probably then we need now to recover the `IDs.txt` event selection pipeline? @DavidDiezIb